doc/rules/phpdoc/phpdoc_tag_no_named_arguments.rst

```diff
@@ -0,0 +1,77 @@
+======================================
+Rule ``phpdoc_tag_no_named_arguments``
+======================================
+
+There must be ``@no-named-arguments`` tag in PHPDoc of a
+class/enum/interface/trait.
```

Comment:
are enum eligible for this tag ?

Reply:
thanks

please consider to add this as a test case, otherwise it may stop working accidentally

Reply:
I think the tag would also make sense for standalone functions (but can be added later, if someone want to contribute it).